src/core/nixl_tcpstore_metadata_backend.cpp

```
@@ -0,0 +1,189 @@
+    client_ = std::make_unique<nixlTcpStoreClient>(
+        host, static_cast<std::uint16_t>(port), std::chrono::milliseconds(timeout_ms));
+    NIXL_DEBUG << "[" << ctx_.getName() << "] connected TCPStore client to " << endpoints;
+}
+
+nixlTcpStoreMetadataBackend::~nixlTcpStoreMetadataBackend() = default;
+
+nixl_status_t
+nixlTcpStoreMetadataBackend::publishKey(const std::string &key, const nixl_blob_t &blob) {
+    try {
+        client_->set(key, blob);
+    }
```

Cppcheck (2.21.0)

[style] The function 'unlock_shared' is never used.

(unusedFunction)